clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/rocksdb/RocksDBSstFileWriter.java

```diff
@@ -77,14 +80,20 @@ public ReusableObjects() {
   private long recordNumInCurrentSSTFile = 0;
   private long recordNumInAllSSTFiles = 0;
   private String fullPathForTempSSTFileDir;
+  private final String fullPathForPartitionDBSnapshot;
   private Optional<Supplier<byte[]>> expectedChecksumSupplier;
   private final String storeName;
   private final int partitionId;
   private final EnvOptions envOptions;
   private final Options options;
   private final boolean isRMD;
+
   private final RocksDBServerConfig rocksDBServerConfig;
 
+  protected Checkpoint makeCheckpoint(RocksDB rocksDB) {
```

Comment:
nit: how about renaming it to `createCheckPoint`? and might also give a comment saying this is testing-only since I see you mocked its output in the unit test